app/services/backgrounds_service.rb

```diff
@@ -1,9 +1,19 @@
 class BackgroundsService < BaseService
+
+  def self.base_url
+    "https://api.unsplash.com"
+  end
+
+  def self.set_api_key(connection)
+    connection.params['client_id'] = ENV['UNSPLASH_ACCESS_KEY']
+  end
+
   def self.search_for_backgrounds(city)
-    response = unsplash_conn.get('/search/photos') do |faraday|
-      faraday.params['query'] = city
-      faraday.params['orientation'] = 'landscape'
+
+    response = connection.get('/search/photos') do |connection|
+      connection.params['query'] = city
+      connection.params['orientation'] = 'landscape'
     end
-    JSON.parse(response.body, symbolize_names: true)
+    format_reponse(response)
   end
 end
```

Comment:
You should get in the habit of placing newlines at the ends of files. Whenver you see that red icon at the end of a file in your PR, that should signal to go fix it. Doing a `git diff` on the command line should show you those missing newlines before you commit, too, with typical git settings.